pkg/api/types.go

```diff
@@ -1737,6 +1737,10 @@ type PodAffinityTerm struct {
 	// for affinity and for RequiredDuringScheduling pod anti-affinity, empty topologyKey is not allowed.
 	// +optional
 	TopologyKey string
+	// The max number of replicas per node/topologyKey to pod anti-affinity;
+	// only available for hard pod anti-affinity (RequiredDuringSchedulingIgnoredDuringExecution)
+	// +optional
+	MaxAntiAffinityTolerant int
```

Comment:
`replicas` is a precise term (things created by a replica set) and doesn't belong here.  If this applies to pods, `MaxPodsPerTopologyKey` would probably be a better description.  What is the effective default for anti-affinity, 1?  Should be documented.

Don't use generic `int`, use `int32`